### PR TITLE
Add scenes to the dutch story.

### PR DESCRIPTION
**Pull request by gewoonwoutje**

### CODE REVIEW

**Comment by gewoonwoutje** [comment]
Please review this, @casey-collab

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: dutch/bos/ontvoering/alleskomtgoed.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Je wordt de volgende dag met een kater wakker in [de hei](../../welkom.md).
```
Comment (casey-collab) [inline dutch/bos/ontvoering/alleskomtgoed.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: dutch/bos/ontvoering/ontvoering.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Terwijl je toekijkt hoe de mannen een gat graven krijg je last van je knieën.
```
Comment (casey-collab) [inline dutch/bos/ontvoering/ontvoering.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: dutch/bos/ontvoering/ontvoering.md (hunk 2 above)
Comment (gewoonwoutje) [inline dutch/bos/ontvoering/ontvoering.md]: I think it's okay now

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: dutch/bos/ontvoering/AllesKomtGoed/alleskomtgoed.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Je wordt de volgende dag met een kater wakker in [de hei](../../welkom.md).
```
Comment (casey-collab) [inline dutch/bos/ontvoering/AllesKomtGoed/alleskomtgoed.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: dutch/bos/ontvoering/ontvoering.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+Voorzichtig probeer je in een andere pose te zitten, maar door het geritsel
+van de struiken trek je de aandacht van de mannen.
+
+Je besluit terug naar [de hei](../../welkom.md) te rennen.
+
+Je besluit te huilen om je moeder terwijl de mannen je afmaken.
+
+De mannen besluiten je te [ontvoeren](alleskomtgoed.md).
```
Comment (casey-collab) [inline dutch/bos/ontvoering/ontvoering.md]: This link is also invalid.

File: dutch/bos/ontvoering/AllesKomtGoed/alleskomtgoed.md (hunk 3 above)
Comment (casey-collab) [inline dutch/bos/ontvoering/AllesKomtGoed/alleskomtgoed.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by gewoonwoutje** [comment]
New working version with a separate directory @casey-collab

**Comment by casey-collab** [comment]
Looks great! :shell:

**Pull request merged**